api/test/wfl/unit/spec_test.clj

```diff
@@ -12,3 +13,13 @@
                         [workloads/wgs-workload-request
                          workloads/aou-workload-request
                          workloads/xx-workload-request])))))
+
+(deftest request-spec-negative-test
+  (testing "Mismatched cram/bam inputs cannot pass spec validation"
+    (letfn [(invalid? [req] (is (not (s/valid? ::spec/workload-request req))))]
+      (let [requests (map #(% (UUID/randomUUID))
+                          [workloads/wgs-workload-request
+                           workloads/xx-workload-request])
+            mismatched-requests (map #(walk/postwalk-replace {:input_cram :input_bam} %)
```

Comment:
`postwalk-replace`?? Looks shiny!

Reply:
thanks 😄  I realized the combination of `rename-keys` and `update-in` is too cumbersome here, thus `postwalk-replace` is a good candidate!